streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -334,18 +342,33 @@ public void needCommit() {
      * @param interval  the interval in milliseconds
      * @throws IllegalStateException if the current node is not null
      */
-    public void schedule(long interval) {
-        if (processorContext.currentNode() == null)
+    public void schedule(final long interval) {
+        if (processorContext.currentNode() == null) {
             throw new IllegalStateException(String.format("%s Current node is null", logPrefix));
+        }
 
         punctuationQueue.schedule(new PunctuationSchedule(processorContext.currentNode(), interval));
     }
 
+    /**
+     * <pre>
+     * - initialize local stores
+     * - registers global stores
+     * - init topology
+     * </pre>
+     */
     @Override
-    public void initTopology() {
+    public void init() {
```

Comment:
Hmm, seems `init()` is also only called once in the constructor of `StreamTask`, and only `initTopology` is reused in `resume`; in that case I'd more prefer to just remove this abstract function and inline that in the constructor (see my comment above).